net.hhoney.rota.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "net.hhoney.rota",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "org.godotengine.godot.BaseApp",
+  "base-version": "3.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "godot-runner",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "rota",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/HarmonyHoney/ROTA.git",
+          "commit": "be542fa2444774fe952ecb22d5056a048399bc25"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/HarmonyHoney/ROTA/releases/download/Flathub/rota-linux.pck",
+          "tag": "Flathub",
```

Comment:
```suggestion
          "url": "https://github.com/HarmonyHoney/ROTA/releases/download/Flathub/rota-linux.pck",
```

Reply:
Whoops, yeah, the tag was supposed to be in the `git` source (just for convenience).